<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsDialog</class>
 <widget class="QDialog" name="StatsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mis Resultados</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filtrar por fecha</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mostrar desde:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEditFilter">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFilter">
        <property name="text">
         <string>Actualizar</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableStats">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>Fecha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Aciertos</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fallos</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblSummary">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Total: 0 Aciertos, 0 Fallos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Cerrar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>StatsDialog</receiver>
   <slot>accept()</slot>
  </connection>
 </connections>
</ui>